### ISSUE TITLE
hashtable arg estimation functions may be wrong

### ISSUE THREAD

**Issue by bocajnotnef** [dib-lab/khmer#1146]
see #1106 #1117 

```
(env)1 bocajnotnef@GCU-Caconym:~/git/khmer⟫ python scripts/normalize-by-median.py -U 83 -x 17 tests/test-data/test-filter-abund-Ns.fq                                                         

|| This is the script 'normalize-by-median.py' in khmer.
|| You are running khmer version 2.0-rc1+53.gf24a2e0.dirty
|| You are also using screed version 0.9
||
|| If you use this script in a publication, please cite EACH of the following:
||
||   * MR Crusoe et al., 2014. http://dx.doi.org/10.6084/m9.figshare.979190
||   * CT Brown et al., arXiv:1203.4802 [q-bio.GN]
||
|| Please see http://khmer.readthedocs.org/en/latest/citations.html for details.


PARAMETERS:
 - kmer size =    32            (-k)
 - n tables =     4             (-N)
 - max tablesize =    17        (-x)

Estimated memory usage is 68 bytes (n_tables x max_tablesize)
--------
*** Warning: The given tablesize is be too small!
*** Estimating false positive rate to be 0.100049378972
making countgraph
DONE with tests/test-data/test-filter-abund-Ns.fq; kept 4 of 4 or 100%
output in test-filter-abund-Ns.fq.keep
Total number of unique k-mers: 3
**
** ERROR: the graph structure is too small for 
** this data set.  Increase data structure size
** with --max_memory_usage/-M.
**
** Do not use these results!!
**
** (estimated false positive rate of 45.698;
max allowable 0.800
```

TL;DR: Estimation funcs say we'll have an FP of  0.100049378972
Script says we have an FP of 45.698

One of those algorithms is horribly wrong. 

~~@ctb @luizirber I have no idea how the math behind any of these functions work--Advice, please?~~

@camillescott  gave enlightenment--pondering required.

**Comment by ctb** [comment]
What was the enlightenment? :)

**Comment by bocajnotnef** [comment]
wiki article on how bloom filters worked.

**Comment by mr-c** [comment]
@bocajnotnef  Is this still an issue?

**Comment by bocajnotnef** [comment]
Unsure--Need to do further investigating.

**Comment by ctb** [comment]
Addressed in #1214.
